tdrs-backend/tdpservice/reports/models.py

```diff
@@ -54,63 +52,53 @@ class Meta:
+    section = models.CharField(
+        max_length=32, blank=False, null=False, choices=Section.choices
+    )
 
     version = models.IntegerField()
 
-    user = models.ForeignKey(User,
-                             on_delete=models.CASCADE,
-                             related_name="user",
-                             blank=False,
-                             null=False)
-    stt = models.ForeignKey(STT,
-                            on_delete=models.CASCADE,
-                            related_name="sttRef",
-                            blank=False,
-                            null=False)
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="user", blank=False, null=False
+    )
+    stt = models.ForeignKey(
+        STT, on_delete=models.CASCADE, related_name="sttRef", blank=False, null=False
+    )
 
     @classmethod
     def create_new_version(self, data):
         """Create a new version of a report with an incremented version."""
         # EDGE CASE
         # We may need to try to get this all in one sql query
         # if we ever encounter race conditions.
-        version = (self.find_latest_version_number(year=data['year'],
-                                                   quarter=data['quarter'],
-                                                   section=data['section'],
-                                                   stt=data['stt']) or 0) + 1
+        version = (
+            self.find_latest_version_number(
+                year=data["year"],
+                quarter=data["quarter"],
+                section=data["section"],
+                stt=data["stt"],
+            )
+            or 0
+        ) + 1
 
-        return ReportFile.objects.create(
-            version=version,
-            **data,
-        )
+        return ReportFile.objects.create(version=version, **data,)
 
     @classmethod
     def find_latest_version_number(self, year, quarter, section, stt):
         """Locate the latest version number in a series of report files."""
-        return self.objects.filter(stt=stt,
-                                   year=year,
-                                   quarter=quarter,
-                                   section=section).aggregate(
-                                       Max("version"))['version__max']
+        return self.objects.filter(
+            stt=stt, year=year, quarter=quarter, section=section
+        ).aggregate(Max("version"))["version__max"]
 
     @classmethod
     def find_latest_version(self, year, quarter, section, stt):
         """Locate the latest version of a report."""
         version = self.find_latest_version_number(year, quarter, section, stt)
 
         return self.objects.filter(
-            version=version,
-            year=year,
-            quarter=quarter,
-            section=section,
-            stt=stt,
+            version=version, year=year, quarter=quarter, section=section, stt=stt,
```

Comment:
Calling `.first()` on the queryset is a safer operation than accessing the first element by index. That way in the event no results are found `None` will be returned instead of an `IndexError`